src/categories/categories.controller.ts

```diff
@@ -17,11 +20,17 @@ export class CategoriesController {
     return this.categoriesService.findAll();
   }
 
+  @Get('getAllApis:id')
```

Comment:
i dont know if there are naming convention for endpoints @PhilemonBrain @Chimexx 

the endpoint name dosnt look nice

Reply:
I have fixed the endpoint name